aip/0192.md

```diff
@@ -0,0 +1,97 @@
+---
+aip:
+  id: 192
+  state: reviewing
+  created: 2019-07-25
+permalink: /192
+redirect_from:
+  - /0192
+---
+
+# Documentation
+
+Documentation is one of the most critical aspects of API design. Users of your
+API are unable to dig into the implementation to understand the API better;
+often, the API surface definition and its corresponding documentation will be
+the only things a user has. Therefore, it is important that documentation be as
+clear, complete, and unambiguous as possible.
+
+## Guidance
+
+In APIs defined in protocol buffers, public comments **must** be included over
+every component (service, method, message, field, enum, and enum value) using
+the protocol buffers comment format. This is important even in cases where the
+comment is terse and uninteresting, as numerous tools read these comments and
+use them.
+
+Services, in particular, **should** have descriptive comments that explain what
+the service is and what users are able to do with it.
+
+**Note:** Many readers will not be native English speakers. Comments **should**
+avoid jargon, slang, complex metaphors, pop culture references, or anything
+else that will not easily translate. Additionally, many readers will have
+different backgrounds and viewpoints; if writing examples involving people,
+prefer people who are non-controversial and no longer alive.
+
+### Style
+
+Comments **should** generally be in gramatically correct American English.
+However, the first sentence of each comment **should** omit the subject:
```

Comment:
and be in the third-person present tense ?

Reply:
Done.